packages/taquito-michel-codec/src/michelson-validator.ts

```diff
@@ -417,6 +420,12 @@ export function assertMichelsonInstruction(ex: Expr): ex is MichelsonCode {
         }
         break;
 
+        case "MIN_BLOCK_TIME":
```

Comment:
I am not sure MIN_BLOCK_TIME should be added here as it doesn't take an argument.